region/maridia/outer.json

```diff
@@ -2108,6 +1688,12 @@
               }
             ]
           }
+        },
```

Comment:
I noticed the Boyon Gate Hall got canSwim reqs on all three runways even though they shouldn't Probably easiest if you fix those.